File: 070.LeetCode@226_Invert_Binary_Tree/070.LeetCode@226_Invert_Binary_Tree.py
class Solution:
    def invertTree(self, root: Optional[TreeNode]) -> Optional[TreeNode]:
        if not root or root.left==root.right:
            return root
        tempright = self.invertTree(root.left)
        templeft = self.invertTree(root.right)
        root.left = templeft
        root.right = tempright
        return root

File: 070.LeetCode@226_Invert_Binary_Tree/070.LeetCode@226_Invert_Binary_Tree.java
class Solution {
    public static TreeNode invertTree(TreeNode root) {
        if (root == null || root.left==root.right) return root;
        TreeNode tempright=invertTree(root.left);
        TreeNode templeft=invertTree(root.right);
        root.right=tempright;
        root.left=templeft;
        return root;
    }
}